lib/cloudantBaseService.ts

```diff
@@ -102,6 +104,14 @@ export default abstract class CloudantBaseService extends BaseService {
     if (!('timeout' in userOptions)) {
       userOptions.timeout = READ_TIMEOUT;
     }
+
+    // @ts-ignore
+    const serviceClass = this.constructor as typeof BaseService;
+    const serviceUrl = userOptions.serviceUrl
+      ? userOptions.serviceUrl
+      : serviceClass.DEFAULT_SERVICE_URL;
```

Comment:
```suggestion
      : BaseService.DEFAULT_SERVICE_URL;
```

Reply:
Nice catch! I fixed it in a7fced3

Reply:
Uh oh, this broke one test https://app.travis-ci.com/github/IBM/cloudant-node-sdk/jobs/556021975#L277 with that. I fixed this in 437fd40